### PR TITLE
MAID-2373 Refactored web hosting manager

### PR DESCRIPTION
**Pull request by shankarsengalani**
Refactored web hosting manager	to have common code as other example apps

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @krishnaIndia

(maidsafe_highfive has picked a reviewer for you, use r? to override)

**Review by joshuef — CHANGES_REQUESTED** [review]
It's a good start, @shankar2105 , but I think we can go further.

`authorisation`, `common` and `initialiser` actions/reducers all seem to be applicable to most _any_ app we'd be building (to me, totally up for debate).

If that is the case we should add all of these to the [common app utils](https://github.com/maidsafe/safe_examples/tree/master/example_common_js_app_utils). 

This also then begs the question of what is the difference with these three reducers/action sets? Do we need to keep them separated semantically? (Could they be merged together?). Whatever the decision on that question, we should try and keep our apps inline for the common code.

Remember, the idea here is to have a set of common actions/reducers so if we're starting a new project, we have the basic functionality in one clear place to draw from. And that we have the same setup across apps (where possible) so other devs can dive into the code easier.

Hope that makes sense, @shankar2105 ? Let me know if you have more Qs

File: example_common_js_app_utils/actions/action_types/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import initialiser from './initialiser';
+
+export default {
+  ...initialiser
```
Comment (joshuef) [inline example_common_js_app_utils/actions/action_types/index.js]: we're missing publicNames here

File: example_common_js_app_utils/reducers/app.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import ACTION_TYPES from '../actions/action_types';
```
Comment (joshuef) [inline example_common_js_app_utils/reducers/app.js]: `app` seems to general for this, I think. Seems more like a `publicNames` reducer or?

**Review by joshuef — CHANGES_REQUESTED** [review]
Good stuff here. Now we're getting towards a lib of common code we can pull in as needed 👍 

Just two minor things here and I think we're good.

**Review by joshuef — APPROVED** [review]

**Pull request merged**